### ISSUE TITLE
Weird URL not working with enabled extension

### ISSUE THREAD

**Issue by Zulcom** [roskomsvoboda/censortracker#112]
The absolutely legitimate link I picked up from email newsletter
```
http://email.mg.buttondown.email/c/eJyNjk2KwzAUg08T72Lejx3bCy-aYXKN4ualjSF1SuIy15-0m24LAoH4hCTRTw7YqRwJCCAAISKQ16h_evztB3DODNg7xsbA_aYvz1rXIutf0dM95UXN0dMVJ2sS2cSTY5FulODFm1E67IxRS5xrfewNnxoaDi15z0suc9qqzuW6HpHa4g6viXLb9fZUNfI4BqKLbf3hrbHWtik4bjkwoBGxCbz6vDm_35yzfFd801-h_77FUaw
```
will lead you to the Lisi Linhart website, but with the enabled extension, it will fail to load the page with `ERR_CONNECTION_REFUSED`.
URL looks weird, but this link works with ext disabled

**Comment by lk-geimfari** [comment]
@Zulcom This URL does not work for me even without an extension.

**Comment by Zulcom** [comment]
Try out `http://costs.buttondown.email/` as an example.

**Comment by lk-geimfari** [comment]
Fixed in 09fcf27
